fdbclient/FDBTypes.h

```diff
@@ -1192,4 +1192,24 @@ struct ReadBlobGranuleContext {
 	bool debugNoMaterialize;
 };
 
+// Store metadata associated with each storage server
+struct StorageMetadataType {
+	constexpr static FileIdentifier file_identifier = 732123;
+	uint64_t createdTime; // comes from Platform::timer_int()
+	bool expireNow;
```

Comment:
nit: better to explain the field usage via code documentation; "expireNow" is a good name for usage but without context could give a wrong impression (again, no strong opinions..)